src/main/java/bio/terra/service/job/JobService.java

```diff
@@ -83,6 +84,7 @@ public JobService(
     this.performanceLogger = performanceLogger;
     this.kubeService =
         new KubeService(appConfig.getPodName(), appConfig.isInKubernetes(), API_POD_FILTER);
+    initialize();
```

Comment:
@pshapiro4broad, so the `initialize` method currently only gets called on startup...so I'm wondering if we might end up initializing stairway multiple times if it's here 🤔

Reply:
If `JobService` is only created via injection aka `AutoWired` (which it should be) then it will only ever be constructed once.